airflow/utils/sqlalchemy.py

```diff
@@ -42,21 +43,21 @@
 
 class UtcDateTime(TypeDecorator):
     """
-    Almost equivalent to :class:`~sqlalchemy.types.DateTime` with
+    Almost equivalent to :class:`~sqlalchemy.types.TIMESTAMP` with
```

Comment:
See previous comment -- this is not the right change, not for Postgresql anyway.